SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -1009,14 +1008,17 @@ void CIncEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contain
   /*--- Define an object to compute the viscous eigenvalue. ---*/
   struct LambdaVisc {
     const bool energy;
-
-    LambdaVisc(bool e) : energy(e) {}
+    const bool species_model;
+    LambdaVisc(bool e, bool species) : energy(e), species_model(species) {}
 
     FORCEINLINE su2double lambda(su2double lamVisc, su2double eddyVisc, su2double rho, su2double k, su2double cv) const {
       su2double Lambda_1 = (4.0/3.0)*(lamVisc + eddyVisc);
       su2double Lambda_2 = 0.0;
       if (energy) Lambda_2 = k / cv;
-      return (Lambda_1 + Lambda_2) / rho;
+      if (species_model)
+        return max(Lambda_1, Lambda_2) / rho;
+      else
+        return (Lambda_1 + Lambda_2) / rho;
```

Comment:
What's the benefit of this?

Reply:
Hi, that implementation appears in the paper of Tomas Economon and in the case of the species model, it turns out that improves the convergence and gives more stability when the species model with the fluid mixture is used.
![image](https://user-images.githubusercontent.com/98025159/193890641-49e9e2ca-ea99-4aa0-aa9e-1a977efcdb4b.png)

Reply:
Keep it for a different PR where you make it the default.

Reply:
Ok, I will add it in the next pull request. Thanks!!